methods.py

```diff
@@ -742,10 +742,12 @@ class ModuleConfigs(Mapping):
             PLATFORM_IDS = ["x86_32", "x86_64"]
             CONFIGURATIONS = ["editor", "template_release", "template_debug"]
             DEV_SUFFIX = ["", ".dev"]
+            DOUBLE_SUFFIX = ["", ".double"]
```

Comment:
I don't think this is needed, it's too niche and will mean that there's 24 (!) variants to choose from.

Handling the option in `build_commandline` like done for `tests` is enough IMO.